torch/csrc/jit/passes/onnx/peephole.cpp

```diff
@@ -620,6 +620,25 @@ void removeMaxPoolUnusedOutput(Block* b) {
   }
 }
 
+void removeBatchNormUnusedInput(Block* b) {
```

Comment:
how about generalizing this pass? We can have a white list of the operator names.